<?xml version="1.0"?>
<launch>

    <arg name="argVehicle"          default="heifu"/>
    <arg name="argID"               default="0"/>
    <arg name="argLimitRollMax"     default="35"/>
    <arg name="argLimitRollMin"     default="-35"/>
    <arg name="argLimitPitchMax"	default="0"/>
    <arg name="argLimitPitchMin"    default="-90"/>
    <arg name="argLimitYawMax"      default="180"/>
    <arg name="argLimitYawMin"      default="-180"/>
    <arg name="argNamespace"        default="$(arg argVehicle)$(arg argID)"/>

    <node name="bev_uav_gimbal_controller_node" type="bev_uav_gimbal_controller_node" pkg="bev_uav_gimbal_controller" output="screen" required="true" ns="$(arg argNamespace)" >
        <param name="paramLimitRollMax"  type="double" value="$(arg argLimitRollMax)"   />
        <param name="paramLimitRollMin"  type="double" value="$(arg argLimitRollMin)"   />
        <param name="paramLimitPitchMax" type="double" value="$(arg argLimitPitchMax)"  />
        <param name="paramLimitPitchMin" type="double" value="$(arg argLimitPitchMin)"  />
        <param name="paramLimitYawMax"   type="double" value="$(arg argLimitYawMax)"    />
        <param name="paramLimitYawMin"   type="double" value="$(arg argLimitYawMin)"    />
    </node>

</launch>

